<!-- -*- mode: XML -*- -->
<launch>

  <arg name="node_name" default="vesc_driver_node" />
  <arg name="filter_size" default="25" />
  <arg name="filter_rate" default="100" />
  <arg name="publisher_rate" default="100" />
  <arg name="max_vel" default="1.0" />
  <arg name="duty_cycle_max" default="1.0" />
  <arg name="duty_cycle_min" default="0.1" />

  <!-- VESC fitler node -->
  <node pkg="vesc_driver" type="vesc_filter" name="vesc" output="screen" >
    <param name="filter_size" value="$(arg filter_size)" />
    <param name="filter_rate" value="$(arg filter_rate)" />
    <param name="publisher_rate" value="$(arg publisher_rate)" />
    <param name="max_vel" value="$(arg max_vel)" />
    <param name="duty_cycle_max" value="$(arg duty_cycle_max)" />
    <param name="duty_cycle_min" value="$(arg duty_cycle_min)" /> 
  </node>

</launch>
